### PR TITLE
Add option to log to filesink

### PR DESCRIPTION
**Pull request by jessicayuen**
Signed-off-by: Jess Yuen <jyuen@lyft.com>

### CODE REVIEW

**Comment by jessicayuen** [comment]
I tested this by starting up the xds-relay server with a bootstrap configuration pointing to a filepath, and saw the logs successfully written there. If we think it's necessary, I can also write an int test for this.

File: internal/app/orchestrator/orchestrator_test.go [hunk 1]
```diff
@@ -55,9 +56,9 @@ func (m mockMultiStreamUpstreamClient) OpenStream(
 
 func newMockOrchestrator(t *testing.T, mockScope tally.Scope, mapper mapper.Mapper,
 	upstreamClient upstream.Client) *orchestrator {
-	logger := log.New("info")
+	logger := log.New("info", os.Stderr)
 	orchestrator := &orchestrator{
-		logger:                logger,
+		logger:                log.New("info", os.Stderr),
```
Comment (LisaLudique) [inline internal/app/orchestrator/orchestrator_test.go:61]: is it worth creating a mocklogger or setting these log levels and stderr as default for testing/no config field provided?

File: internal/pkg/log/zap.go [hunk 2]
```diff
@@ -12,14 +13,18 @@ type logger struct {
 }
 
 // New returns an instance of Logger implemented using the Zap logging framework.
-func New(logLevel string) Logger {
+//
+// logLevel is mandatory, can be one of DEBUG, INFO, WARN, ERROR, FATAL.
+// outputPath is mandatory. Use os.Stderr if unsure.
```
Comment (LisaLudique) [inline internal/pkg/log/zap.go:18]: nit: writeTo

File: internal/app/orchestrator/orchestrator_test.go (hunk 1 above)
Comment (jessicayuen) [reply internal/app/orchestrator/orchestrator_test.go:61]: that's a good idea, I can do it as a part of this PR.

**Pull request merged**